src/types/messages.rs

```diff
@@ -121,6 +121,13 @@ impl OrderMetadataAndMessage {
     }
 }
 
+#[derive(serde::Serialize)]
+pub struct ProcessOrderResponse {
+    pub message: String,
```

Comment:
maybe this, since the message should be some pre-defined string
```suggestion
    pub message: &'static str,
```